### DISCUSSION TITLE
关于搜索，视图的建议

### DISCUSSION THREAD

**Discussion by JeremyMin666** [spencerwooo/onedrive-vercel-index#373] [Ideas]
1.搜索建议输入过密码（login 过）才显示在搜索结果中。

2.搜索报错, 
Application error: a client-side exception has occurred (see the browser console for more information).

3. 建议增加：禁用搜索功能，如果无法做到1. 

4.搜索无法搜索中文

*5. 建议对于每个文件夹记忆list 和 grid视图，不然不同文件夹还得每次手动切好麻烦！！

谢谢大佬无私付出！

**Comment by spencerwooo** [comment]
- 1 + 3 - https://github.com/spencerwooo/onedrive-vercel-index/pull/283#discussion_r790121386 and https://github.com/spencerwooo/onedrive-vercel-index/discussions/356
- 2 - 应该跟 https://github.com/spencerwooo/onedrive-vercel-index/discussions/353 一样的问题？如果能复现的话。
- 4 - 对 CJK 的搜索已经有很大的改进了（[#354](https://github.com/spencerwooo/onedrive-vercel-index/pull/354)），你可以试一下 Demo 项目搜索「申鹤」「原理」之类，如果你自己的项目搜不出来很可能是 OneDrive 那边没有建立好搜索索引，可以等个几天再试试。有关搜索不靠谱的问题，因为很大程度都是上游 OneDrive API 的问题，所以本项目除非是极为离谱的 bug，都不会在考虑处理了。
- 5 - 可以但是……没必要吧。

搜索功能禁用可以考虑，但是我觉得其实并没有什么「隐私泄露」，本项目的密码保护还是请不要太依赖，还是那句话 ——「防君子不防小人」。
